common/CommunityToolkit.App.Shared/Renderers/ToolkitDocumentationRenderer.xaml.cs

```diff
@@ -168,17 +168,6 @@ private static async Task<string> GetDocumentationFileContents(ToolkitFrontMatte
 
         var path = metadata.FilePath;
 
-        if (isSingleExperimentHead || isProjectTemplateHead)
```

Comment:
Oh did you fix the Uno issue here too? Can we close that issue in our repo?

Reply:
I didn't make this change on this branch. See latest merged commit.